controllers/idler/idler_controller_test.go

```diff
@@ -143,8 +143,9 @@ func TestEnsureIdling(t *testing.T) {
 		reconciler, req, cl, allCl, dynamicClient := prepareReconcile(t, idler.Name, getHostCluster, idler, nsTmplSet, mur)
 
 		podsTooEarlyToKill := preparePayloads(t, reconciler, idler.Name, "", freshStartTimes(idler))
-
+		podCrashLooping := preparePayloadCrashloopingPod(t, reconciler, idler.Name, "restartCount-")
```

Comment:
Let's also add pods with some restart count which is lower than the threshold to make sure they are not deleted. And add some crashlooping controller (in addition to the standalone pods). Like a Deployment. So we can cover more use cases.

Reply:
added in [576b44c](https://github.com/codeready-toolchain/member-operator/pull/630/commits/576b44c40a3fbd763c7792142a84d8a245b83b23)